### ISSUE TITLE
Does not annotate directive.controller

### ISSUE THREAD

**Issue by btford** [btford/ngmin#21]
The directive.controller injection does not catch. See example:

``` javascript
  angular.module('some-module').directive('some-directive', function () {
    return {
      restrict: 'A',
      controller: function($scope) {
        this.scope = $scope;
      },
    …
});
```

(from btford/grunt-ngmin#4, via @evjinn)

**Comment by alexgorbatchev** [comment]
:+1:

**Comment by btford** [comment]
Landed in 6c2b10c34eb893bd39697d9c4b53b68cba2de36a, and now available in `v0.3.5`. Please let me know if you have issues. Thanks!

**Comment by alexgorbatchev** [comment]
Awesome, thank you very much!
